config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
Probably you've not intended this?
I guess we should leave this `{{.WorkerCount}}` or we become unable set desired capacity